### ISSUE TITLE
[BUG] No rangecheck, bot crashes with large messages

### ISSUE THREAD

**Issue by sporkwitch** [Dragory/modmailbot#72]
`Error: DiscordRESTError [50035]: Invalid Form Body`
`  content: Must be 2000 or fewer in length.`

The bot should perform a rangecheck on the message before relaying it, as the insertion of the sender's name can cause it to overrun, the API error then crashing the bot.

Rangecheck should be of the form $MESSAGE ≤ 2000 - len(senderNameString)

If I have time tomorrow, I'll look over the code and see if I can add it, then submit a pull request.  Let me know if you beat me to it (it'll take me longer, since I haven't looked over the code and don't know JS)

**Comment by Dragory** [comment]
Good catch. I imagine the best solution would be to split the message into 2 messages if it goes over the limit (but still keep it as 1 message in the logs, since those are not limited by discord's limitations). I'll look into it.

**Comment by sporkwitch** [comment]
@Dragory Honestly more work than it's worth, Discord could do that themselves if they wanted.  I'd just have the bot reply with an error message saying it's too long.  Heck, wouldn't necessarily even need to do the range check, let discord handle that, catch the exception, and handle it by passing along the error (does javascript have exception handling? I honestly don't know lol)
